extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -71,7 +71,7 @@ export const EXPANDABLE_COMPONENTS = {
     '0 78.62 23"/></g></svg>',
     actionIcon: ActionIcon.EXPAND,
     localizedStringId: LocalizedStringId.AMP_STORY_TOOLTIP_EXPAND_TWEET,
-    selector: 'amp-twitter',
+    selector: 'amp-twitter[interactive=yes]',
```

Comment:
I think we would want this to equally apply to all embeds.  Is there a way that we don't have to repeat this every time we add a new embed?

Reply:
I imagine we could dynamically append `[interactive=true]` at the end of the selectors automatically, but at this point I think having it here is easier (still one line), whereas one more layer of looping through and editing the config when there already are many really scares me :(